test-result-summary-client/src/Dashboard/Widgets/Graph/Dacapo.jsx

```diff
@@ -14,29 +14,30 @@ const map = {
 	"dacapo-jdk8": "Test_openjdk8_j9_sanity.perf_x86-64_linux"
 };
 
-let display = {
-	"dacapo-jdk11": true,
-	"dacapo-jdk8": true,
-};
+let servers = ['AdoptOpenJDK', 'CustomizedJenkins'];
```

Comment:
We should not hardcode the value. The value should from the server-side. Hardcode the server value means we have to maintain and update the list.